cli/src/config-schema.json

```diff
@@ -164,7 +164,8 @@
                     "description": "Conflict marker style to use when materializing conflicts in the working copy",
                     "enum": [
                         "diff",
-                        "snapshot"
+                        "snapshot",
+                        "git-diff3"
```

Comment:
(Bikeshedding) For the name, I'd prefer if "git" and "diff3" were synonyms. Either is easier to remember that "git-diff3".

Reply:
Maybe we could just have it be called "git", since it seems unlikely we'd support anything other than "diff3" from Git. If we were to use "diff3" as-is, we'd probably want to rename "diff" to something else to avoid confusion since they're pretty different.

Reply:
We could also consider renaming "diff" to "default" or "jj" or "side-and-diffs".  Git supports several merge styles, so I think supporting "diff3" is good if we can (the [other ones](https://git-scm.com/docs/git-merge#_how_conflicts_are_presented) are either lossy or confusing, we shouldn't support them, but they do exist). Supporting "git" as a synonym makes sense to me as well, though I could also do without it.

I'd love it if we came up with better names before merging this, it might be hard to change later. It is, however, tricky to come up with something better. If we don't have a better idea, we should probably just merge instead of bikeshedding forever.

Reply:
I suppose another argument in favor of calling it "diff3" rather than "git" is because those conflict markers originally come from the "diff3" program (as far as I know).

Right now, I think I like "default" more than "jj" because calling it "jj" seems to imply that the other conflict marker styles aren't Jujutsu, even though "snapshot" is also Jujutsu-specific currently.

If we replaced "snapshot" with an extended version of "diff3", then we could probably get away with having "jj", "diff3", and "diff3+" as you suggested in your other comment.

Reply:
"snapshot", "diff", and "git" sound good to, but I don't feel strongly

Reply:
> "snapshot", "diff", and "git" sound good to, but I don't feel strongly

I think this is my current favorite option. After thinking about it a bit more, I don't love "diff3" as a name because it sounds like it should show a diff, but it doesn't actually show any diffs, which is confusing. Since we have a format that actually does show diffs, it seems like users could get confused about which is which.